exist-core/src/main/java/org/exist/xquery/functions/fn/CollatingFunction.java

```diff
@@ -22,10 +22,7 @@
 package org.exist.xquery.functions.fn;
 
 import com.ibm.icu.text.Collator;
-import org.exist.xquery.Function;
-import org.exist.xquery.FunctionSignature;
-import org.exist.xquery.XPathException;
-import org.exist.xquery.XQueryContext;
+import org.exist.xquery.*;
```

Comment:
Do we not prefer single imports?

Reply:
pooh, different context I see different 'rules'

Reply:
this is std IntelliJ